whiteBoarding/splitMiddleStr/keg-splitMiddleStr.js

```diff
@@ -0,0 +1,13 @@
+// Here's a start...10-28-18
+
+str1 = abcdefghijklmnopqrstuvxyz
```

Comment:
It probably would have error out here, but hoisting, ya know?

Let's check it out, yup -> `ReferenceError: abcdefghijklmnopqrstuvxyz is not defined`

Reply:
needs to actually be a string